composer.json

```diff
@@ -27,7 +27,8 @@
     "autoload": {
         "psr-4": {
             "Http\\Discovery\\": "src/"
-        }
+        },
+        "files": ["src/functions_include.php"]
```

Comment:
is there no autoloading for functions?

Reply:
I think only classes are autoloaded.

Reply:
right, otherwise we would need a naming scheme for function to filename...